Core/src/main/java/com/plotsquared/core/command/Visit.java

```diff
@@ -326,6 +317,29 @@ public CompletableFuture<Boolean> execute(
         return CompletableFuture.completedFuture(true);
     }
 
+    private boolean isInvalidPageNr(String arg) {
+        if (MathMan.isInteger(arg)) return false;
```

Comment:
I think the code style requires all ifs etc. these have the braces applied

Reply:
Ok. I have changed it.